scm/etc/scripts/scm_read_obs.py

```diff
@@ -6,6 +6,71 @@
 import math
 import forcing_file_common as ffc
 
+def read_MOSAiC_obs(obs_file, time_slices, date):
```

Comment:
I know this is consistent with the existing code, but [PEP8 standards call for four spaces of indentation at each level](https://peps.python.org/pep-0008/#indentation). At the very least indentation should be consistent at all levels, and this mixes 2- and 4-space indentation. Can we change this to use all 4 spaces here and think about changing the rest of the script later?

Reply:
We're not supporting these old plotting scripts. I, and maybe a few other people, use them, but I don't think that they're worth hardening at this point. I've updated the indentation, but I hope that we can remove this from the repo at some point and replace with something better (or not replace at all).